src/route/ProfessorAccount.tsx

```diff
@@ -26,13 +29,32 @@ export default function ProfessorAccount() {
     handleConfirmedPasswordChange('');
   };
 
+  // get both passwords, compare them, if they are identical and not null return true, otherwise return false
+  const verifyPassword = (
+    passwordText: string,
+    repeatedPasswordText: string,
+  ): boolean => {
+    // create a new regex
+    const regex = new RegExp(
+      /^(?=.*[A-Za-z])(?=.*\d)(?=.*[@$!%*#?&])[A-Za-z\d@$!%*#?&]{8,}$/,
+    );
+
+    // if regex is tester true and passwords are identical
+    if (regex.test(passwordText) && passwordText === repeatedPasswordText) {
+      return true;
+    }
+    return false;
+  };
+
   const submitForm = (event: any) => {
+    setPasswordErrored(true);
     // close the overlay
-    setIsOpenOverlay(false);
+    const isVerified = verifyPassword(password, confirmedPassword);
+    if (isVerified === true) {
```

Comment:
ça revient à dire true === true 
```suggestion
    if (isVerified) {
```